### PR TITLE
Continuation of ssl verification modifications for use in non-polling mode

### PR DESCRIPTION
**Pull request by rstevens011**
*Description of changes:*
In #46 I updated the logic for polling mode but did not update the logic for non-polling mode.  When in this mode, it appears that crhelper.util's `_send_response` function will call CFN directly.  This extends the existing ssl_verify argument passed in from the initial entry point in resource_helper.py to the HTTPSConnection's SSLContext.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rstevens011** [comment]
The test reduction check issues are basically the result of changing the ca bundle path potentially.  Normally, this code path would not be used, except for cases where endpoints are signed by a CA not normally trusted by the Lambda's runtime environment.

@jaymccon

File: crhelper/utils.py [hunk 1]
```diff
@@ -22,9 +26,18 @@ def _send_response(response_url, response_body):
     split_url = urlsplit(response_url)
     host = split_url.netloc
     url = urlunsplit(("", "", *split_url[2:]))
+    ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
+    if isinstance(ssl_verify, str) and path.exists(ssl_verify):
+        ctx.load_verify_locations(cafile=ssl_verify)
+    else:
+        logger.warning("Cert path {0} does not exist!.  Falling back to using system cafile.".format(ssl_verify))
```
Comment (jaymccon) [inline crhelper/utils.py:33]: this fires if `ssl_verify=False` was the intent to have this warning only when `ssl_verify=path` and path does not exist?

Comment (rstevens011) [reply crhelper/utils.py]: I've updated this logic in the latest commit

**Comment by rstevens011** [comment]
thanks!

**Pull request merged**